go-controller/pkg/libovsdb/ops/switch.go

```diff
@@ -328,11 +328,17 @@ func createOrUpdateLogicalSwitchPorts(nbClient libovsdbclient.Client, sw *nbdb.L
 	return err
 }
 
-// CreateOrUpdateLogicalSwitchPortsOnSwitchOps creates or updates the provided
+// CreateOrUpdateLogicalSwitchPortsOnSwitchOpsWithCustomFields creates or updates the provided
 // logical switch ports, adds them to the provided logical switch and returns
 // the corresponding ops
-func CreateOrUpdateLogicalSwitchPortsOnSwitchOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation, sw *nbdb.LogicalSwitch, lsps ...*nbdb.LogicalSwitchPort) ([]libovsdb.Operation, error) {
-	return createOrUpdateLogicalSwitchPortsOps(nbClient, ops, sw, false, lsps...)
+func CreateOrUpdateLogicalSwitchPortsOnSwitchOpsWithCustomFields(nbClient libovsdbclient.Client, ops []libovsdb.Operation, sw *nbdb.LogicalSwitch, customFields []ModelUpdateField, lsps ...*nbdb.LogicalSwitchPort) ([]libovsdb.Operation, error) {
```

Comment:
nit: `CreateOrUpdateLogicalSwitchPortsOnSwitchWithCustomFieldsOps`

sorry, difficult to realize this little things.

Reply:
np, [fixed](https://github.com/ovn-kubernetes/ovn-kubernetes/compare/1ef70b6dbe945653ac30d7aee15ff6613655267f..414d3e5454164ced40cde6f78213c091706ebe73).